### DISCUSSION TITLE
Is there any way to customize WYSIWYG in Directus ?

### DISCUSSION THREAD

**Discussion by SimonSemaanAtos** [directus/directus#14808] [Archive]
If I want to add a custom tag in the editor and add it to the toolbar, like when i click on the button I add a <component></component> with some attributes, is there any way to do that ?

**Comment by br41nslug** [comment]
There's currently no way to extend the built-in wysiwyg interface besides the provided options with the interface. I think you can do something like you describe using the custom formats but im not completely sure.
![image](https://user-images.githubusercontent.com/9389634/182565677-d3b8f5f7-05f9-45bd-b6cf-d87993ee4615.png)

**Reply by SimonSemaanAtos** [reply]
Thank you for your help !
